package.json

```diff
@@ -0,0 +1,157 @@
+{
+  "name": "trampss-todomvc",
+  "version": "0.1.0",
+  "description": "TodoMVC with React / Redux / Redux-Saga",
+  "license": "MIT",
+  "scripts": {
+    "start": "cross-env NODE_ENV=development webpack-dev-server --host 0.0.0.0 --inline --hot --port=3000 --history-api-fallback --watch-poll=1000",
+    "build": "cross-env NODE_ENV=production webpack -p",
+    "lint": "eslint --cache --ext js,jsx src",
+    "test": "jest",
+    "analyze": "webpack --json > bundle.analyze.json"
+  },
+  "dependencies": {
+    "babel-plugin-transform-inline-environment-variables": "^0.0.2",
+    "babel-polyfill": "^6.20.0",
+    "hoc-little-router": "^1.0.0",
+    "isomorphic-fetch": "2.2.1",
+    "lodash": "^4.17.4",
+    "prop-types": "^15.5.8",
+    "react": "next",
+    "react-dom": "next",
+    "react-hot-loader": "3.0.0-beta.6",
+    "react-motion": "^0.5.0",
+    "react-redux": "^5.0.2",
+    "recompose": "^0.22.0",
+    "redux": "^3.6.0",
+    "redux-little-router": "^13.1.2",
+    "redux-saga": "^0.14.3",
+    "todomvc-app-css": "^2.1.0",
+    "trampss-redux-factory": "^3.5.0",
+    "trampss-redux-saga-tester": "^1.0.0"
+  },
+  "devDependencies": {

```

Comment:
I have the feeling that dependencies can be stripped out : 
 - remove all dependencies
Then : 
 - run test / build / lint 
 - add missing dependency
 - loop